roles/gitea/tasks/main.yaml

```diff
@@ -7,7 +7,7 @@
     state: present
 
 
-- name: Install Gitea Server
+- name: "Install Gitea Server"
```

Comment:
No need to quote unless there is a variable

Reply:
There's not a need, but it's more consistent and makes it easy to add a variable in later.

As a personal thing, my syntax highlighting detects these files as YAML rather than Ansible specifically, so highlights quoted vs. unquoted strings differently; different editors have different settings so let's aim for something that works generally.